docs/querying/multi-value-dimensions.md

```diff
@@ -61,20 +77,81 @@ By default, Druid sorts values in multi-value dimensions. This behavior is contr
 
 See [Dimension Objects](../ingestion/ingestion-spec.md#dimension-objects) for information on configuring multi-value handling.
 
+### SQL-based ingestion
+Multi-value dimensions can also be inserted with [SQL-based ingestion](../multi-stage-query/index.md). The multi-stage query engine does not have direct handling of class Druid multi-value dimensions. A special pair of functions, `MV_TO_ARRAY` which converts multi-value dimensions into `VARCHAR ARRAY` and `ARRAY_TO_MV` to coerce them back into `VARCHAR` exist to enable handling these types. Multi-value handling is not available when using the multi-stage query engine to insert data.
```

Comment:
"classic" (spelling)

Although… what does it mean to say that MSQ doesn't have "direct handling of classic Druid multi-value dimensions"? I would think it does directly handle them, if you use `ARRAY_TO_MV`? I guess I'm not sure what you're trying to say here.

Grammar for the sentence starting with "A special pair of functions" is kind of wonky. Please rewrite it to be clearer.

Reply:
I guess I was thinking of `groupByEnableMultiValueUnnesting` which looking at the code is actually allowed by default, and it is the web-console which sets it to false for MSQ queries by default. I'll try to clarify stuff